packages/extractor/src/extractors/YoutubeExtractor.ts

```diff
@@ -46,6 +46,8 @@ export class YoutubeExtractor extends BaseExtractor<YoutubeExtractorInit> {
             this._ytLibName = name;
         }
 
+        process.emitWarning("YoutubeExtractor uses scraping-based streaming libraries which is known to be unstable at times.\nAn alternative is to use discord-player-youtubei https://github.com/retrouser955/discord-player-youtubei")
```

Comment:
```suggestion
        process.emitWarning("YoutubeExtractor uses scraping-based streaming libraries which is known to be unstable at times. An alternative is to use discord-player-youtubei https://github.com/retrouser955/discord-player-youtubei");
```

Could we also link the github issue here?